client/views/omnichannel/currentChats/FilterByText.tsx

```diff
@@ -74,11 +76,46 @@ const FilterByText: FilterByTextType = ({ setFilter, reload, ...props }) => {
 	const Tags = useCurrentChatTags();
 
 	const onSubmit = useMutableCallback((e) => e.preventDefault());
-	const reducer = function (acc: any, curr: string): any {
-		acc[curr] = '';
-		return acc;
+
+	const handleCustomFieldsFilterToggle = (): void => setCustomFieldsSearchBarOpen(true);
+
+	const { value: allCustomFieldsDefinition, phase: stateCustomFields } = useEndpointData('livechat/custom-fields');
+
+	const jsonConverterToValidFormat = (customFields: Omit<ILivechatCustomField, '_updatedAt'>[]): CustomFieldsData => {
```

Comment:
If possible, can we move this outside of comp into its own function?

Reply:
Sure, its done, please check